webapp/src/simulatorDisplay.tsx

```diff
@@ -0,0 +1,74 @@
+
+
+        return (
+            <div
+                className='simulator-display'
+                id="wrap"
+                style={{
+                    transform: `scale(${scale})`,
+                    transformOrigin: `0 0`,
+                    width: this.scalePixel(368),
+                    height: this.scalePixel(432.938),
+                }}
+            >
+                <div className='debug-stats' id="debug-stats"></div>
+                <div className='screen-back' id="screen-back"></div>
+                <div id="screen">
+                    <div
+                        id="paint-surface-container"
+                        style={{
+                            top: this.scalePixel(12),
+                            left: this.scalePixel(12),
+                        }}
+                    >
+                        <canvas
+                            id="paint-surface"
+                            width="160"
+                            height="120"
+                            className="borders"
+                            style={{
+                                transform: `scale(${scale})`,
+                                transformOrigin: `0 0`,
+                                height: 229,
+                                width: 312,
+                                left: this.scalePixel(28),
+                                top: this.scalePixel(20),
+                                clipPath: `polygon(0px 5.33333px, 5.33333px 0px, 338.667px/1.2) 0px, 344px 5.33333px, 344px 256.667px, 338.667px 262px, 5.33333px 262px, 0px 256.667px)`,
```

Comment:
Any time you have a bunch of fixed values, try to provide a comment and some insight into how they were picked